A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -3320,6 +3326,64 @@ class FMCMainDisplay extends BaseAirliners {
     isAltitudeManaged() {
         return SimVar.GetSimVarValue("AUTOPILOT ALTITUDE SLOT INDEX", "number") === 2;
     }
+
+    getTransitionAltitude(icao, phase) {
+        NXApi.getAirport(icao)
+            .then((data) => {
+                this.transitionAltitude = data.transAlt;
+                if (this.transitionAltitude === -1) {
+                    if (phase === "origin") {
+                        SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", 18000);
+                        return;
+                    }
+                    if (phase === "destination") {
+                        SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", 18000);
+                        return;
+                    }
+                } else {
+                    if (phase === "origin") {
+                        SimVar.SetSimVarValue("L:AIRLINER_TRANS_ALT", "Number", this.transitionAltitude);
+                        return;
+                    }
+                    if (phase === "destination") {
+                        SimVar.SetSimVarValue("L:AIRLINER_APPR_TRANS_ALT", "Number", this.transitionAltitude);
+                        return;
+                    }
+                }
+            });
+    }
+
+    // Only update when changing origin & destination airport
+    updateDepartArrive() {
+        if (this.flightPlanManager.getOrigin() && this.flightPlanManager.getOrigin().ident) {
+            if (this.flightPlanManager.getDestination() && this.flightPlanManager.getDestination().ident) {
+                const originAirport = this.flightPlanManager.getOrigin().ident;
+                const destinationAirport = this.flightPlanManager.getDestination().ident;
+                if (this.offlineTACore.offline !== true) {
+                    if (this.currentOrigin !== originAirport) {
+                        NXDataStore.set("PLAN_ORIGIN", originAirport);
```

Comment:
This is no longer required, is it?

Reply:
I think it isn't.
That NXDatastore need to send telex isn't? I thought that use for add departure / destination in our map.

@nistei 
Nistei you need this?
When spawn on runway, datastore origin  & destination is missing so i added it.

If you don't have to use that, i'll remove it

Reply:
I'm not sure what this has to do with me. I suggested this since you've mentioned you needed origin and destination when spawning on the runway.

Reply:
Sounds like we can drop it then.

Reply:
Ok remove it